src/shared/file_op.c

```diff
@@ -884,28 +884,38 @@ int MergeAppendFile(const char *finalpath, const char *files, const char *tag, i
     }
 
     finalfp = fopen(finalpath, "a");
-    if (!finalfp) {
+    if (finalfp == NULL) {
         merror("Unable to append merged file: '%s' due to [(%d)-(%s)].", finalpath, errno, strerror(errno));
         return (0);
     }
 
     fp = fopen(files, "r");
-
-    if (!fp) {
-        merror("Unable to merge file '%s' due to [(%d)-(%s)].", files, errno, strerror(errno));
+    if (fp == NULL || (fseek(fp, 0, SEEK_END) != 0)) {
```

Comment:
I thing the `fseek` should be in a different `if` after this one.

Reply:
Done